### ISSUE TITLE
Crash after updating to BETA14 and 15

### ISSUE THREAD

**Issue by TheSpicePhantom** [Azanor/thaumcraft-beta#895]
[Gist with crashlog and server log](https://gist.github.com/TheSpicePhantom/d6bf5993e8c20d3274b7d8062f73ae95)

All i can say is that I use forge 14.23.4.2703 and thaumcraft crashes after arriving at the aura thread.
Modpack is https://minecraft.curseforge.com/projects/mmd-public-pack-reloaded

We tried updating from BETA13 to BETA 15.

**Comment by yunruiwatts** [comment]
_Not_ a duplicate of the crash everyone else is reporting!

**Comment by StangleKhan** [comment]
Getting the exact same crash. 

[crash-2018-05-27_11.47.58-server.txt](https://github.com/Azanor/thaumcraft-beta/files/2042775/crash-2018-05-27_11.47.58-server.txt)

Using a custom modpack for a private server.

**Comment by yunruiwatts** [comment]
@StangleKhan looks like the stack trace for your crash is not the same. It might not be caused by the same thing, but yours is also not a duplicate of the bug that everyone else is posting.

**Comment by StangleKhan** [comment]
Ah I'll post it somewhere else then

**Comment by enderger** [comment]
Possible duplicate of #893

**Comment by TheSpicePhantom** [comment]
Def Not a duplicate of #893

**Comment by yunruiwatts** [comment]
I wouldn't say it's _definitely_ not a duplicate, but the stack traces are different.

**Comment by edhelday** [comment]
I don't know if my case it's duplicated. I have tried update from 13 to 15, it crashed and I tried update to 14 with no problem. Then tried from 14 to 15 and crashed. There's the log:
[crash-2018-05-28_01.17.20-client.txt](https://github.com/Azanor/thaumcraft-beta/files/2043138/crash-2018-05-28_01.17.20-client.txt)
I have a few mods: Forge, Optifine, Journeymap, Baubles and Thaumcraft, and play only one player mode.
Do you think if I load the game at beta 14 and wipe all Thaumcraft content, can it update to 15?

**Comment by yunruiwatts** [comment]
@edhelday no, you won't be able to update to 15 - Thaumcraft content should be considered broken in 15

**Comment by edhelday** [comment]
I see, it's impossible to wipe out all thaumcraft content, like subterranian crystals, ambar and overworld quartz, magic forest, tainted zones... it's a madness, better re-begin at same seed.

**Comment by enderger** [comment]
How to remove thaumcraft content
1: uninstall thaumcraft
2: load the world
3: install the new beta
4: enable post-creation worldgen

**Comment by edhelday** [comment]
Thanks enderger!

**Comment by enderger** [comment]
No problem

**Comment by edhelday** [comment]
Ok, Open chest without Thaumcraft have no influence, the chest stay empty when installing TC, opened before or not.
The chest than maintain its content are exclusively shulkers with TC items inside AND stored into other chest.
Unfortunately the game crashed when I go to tainted zone

**Comment by enderger** [comment]
Check thaumcraft 6’s Minecraft fourm post

**Comment by enderger** [comment]
Unfortunately, two things:
1: Thaumcraft items are lost when using this method, as it is essentially telling the items that their ids can’t be found in Minecraft, hence forcing them to not load(but may glitch the world, as esswntially they had to be removed to let the game run)
2: Thaumcraft is im beta, so the mechanic is probably a bit finicky. To improve future versions, reporting bugs in the post-creation worldgen would go a long way. It may be possible to use the code again in the future to fix the bugs in it.

**Comment by enderger** [comment]
(https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1292130-thaumcraft-6-1-beta15) Forum URL

Removing thaumcraft basically forces the game to not be able to load the items, and hence delete them.

**Comment by enderger** [comment]
Probably bug in taint loading: if beta15 is used, it is just good old bug #858 crashing due to tc particles.

**Comment by enderger** [comment]
It was B15, they rolled the pack back to the previous version.

**Comment by enderger** [comment]
I recommend you to do the same

**Comment by enderger** [comment]
Or wait for the update later today or tomorrow

**Comment by enderger** [comment]
Realises that the thread creator is one of the pack authors: oh, hi. How’s the update going?

**Comment by enderger** [comment]
B13 plays nice with Linux, but B14 crashes for me. TC6 B14 may not play nice with Linux.

**Comment by Morpheus1101** [comment]
B14 TC6 caused cascading worldgen untill server crashed and as TheSpicePhantom said, B15 dosent play nice either

**Comment by edhelday** [comment]
REPORT

Since my last message I stored (with beta 13) all TC items in shulkers and the shulkers in my ender chest,
then I uninstalled TC beta and enter game so all TC content are gone (my tainted zone was turned into a huge hole, but the light don't pass through, so something lasts anyway) but i do not touch the shulkers
Then, I installed beta 14, and entered game, exit, installed beta 15...

At this point the TC content like crystals has returned, but attempts to access tainted zone crashed game.
Not touched shulkers yet.
And waited for beta 16.

Today, I have installed beta 16. I have open all shulkers and get all TC items without any problem, so I saved all my goods. The tainted zone is accesible again, but auras may had set to zero when deinstalled TC, so tainted blocks are slowly turning dirt and porous stone, and all taint mobs had gone.
However i have lost all investigation, so i have begin the Thaumonomicon from zero. (logical and acceptable when all other resources and buildings are saved)

(I had a huge tainted zone because I searched the taller mountain i could find, and placed an essence distillery at bedrock height, due to the taintacles and taint appears on the top of the tainting place, so I have time to distil many, many materials until taint reach the subsoil distillation place)
